lib/errors/kuzzleError.js

```diff
@@ -6,11 +6,15 @@ const util = require('util');
  * @constructor
  */
 class KuzzleError extends Error {
-  constructor (message, status) {
+
+  constructor(message, status, errorName = 'Undocumented error', code = 0) {
     super(message);
 
     this.status = status;
 
+    this.code = code;
+    this.errorName = errorName;
```

Comment:
this is an error class, so obviously, the name is the error's name

```suggestion
    this.name = name;
```

Reply:
same as above